src/ParameterDistributions.jl

```diff
@@ -412,11 +413,10 @@ function ParameterDistribution(
 
     # flatten the structure
     distribution_vec = [distribution]
-    flat_constraint_vec = cat(constraint_vec..., dims = 1)
     name_vec = [name]
 
     # build the object
-    return ParameterDistribution(distribution_vec, flat_constraint_vec, name_vec)
+    return ParameterDistribution(distribution_vec, constraint_vec, name_vec)
```

Comment:
I think the input arguments to this function ensures that `constraint_vec` is already a flat vector, so `cat(..., dims=1)` shouldn't be needed.